azure.yaml

```diff
@@ -1,34 +1,80 @@
-# yaml-language-server: $schema=https://raw.githubusercontent.com/Azure/azure-dev/main/schemas/v1.0/azure.yaml.json
+trigger:

```

Comment:
you are overwriting the azd azure.yaml file. don't do that. 
Instead create an ado pipeline configuration file like this:
.azdo/pipelines/app-ci.yaml

Reply:
done